pom.xml

```diff
@@ -44,6 +44,7 @@
       <hawkbit.version>0.3.0M7</hawkbit.version>
       <aws.sdk.version>1.11.415</aws.sdk.version>
       <gcp.sdk.version>0.77.0-alpha</gcp.sdk.version>
+      <mongodb.driver.version>4.0.6</mongodb.driver.version>
```

Comment:
MongoDB driver version is already managed by Spring. So rather go for just setting `<mongodb.version>4.0.6</mongodb.version>` without duplication the dependencies below. Maybe you can also leave a comment that this could be removed with a newer Boot version (cf. [here](https://github.com/eclipse/hawkbit/blob/master/pom.xml#L141-L143))

Reply:
Good catch, thanks for the hint.